src/BitClout.ts

```diff
@@ -11,6 +11,32 @@ export class BitClout {
     this.baseUrl = baseUrl;
   }
 
+  async getIsHodlingPublicKey({
+    publicKey,
+    isHodlingPublicKey,
+  }: {
+    publicKey?: string;
+    isHodlingPublicKey?: string;
```

Comment:
If both of these are required fields we can have typescript enforce at build time (and auto document code) by removing the `?`s

```suggestion
    publicKey: string;
    isHodlingPublicKey: string;
```